suite-native/navigation/src/navigators.ts

```diff
@@ -18,10 +19,15 @@ import {
     AddCoinAccountStackRoutes,
 } from './routes';
 
+type AddCoinFlowParams = RequireAllOrNone<
+    { networkSymbol: NetworkSymbol; accountType: AccountType; accountIndex: number },
```

Comment:
Is `accountIndex` necessary to send via route parameter? You could easily get it from a new selector that would select all accounts of the network and account type and returned index of last account + 1.

Reply:
If I do that it can happen, that the screen rebuilds and then the selector would return the new account as a last one and I would be waiting for another -> infinite loading.

For example, there is useState in `AccountDetailScreen`

This makes the selection of account idempotent